ui/app/templates/allocations/allocation/index.hbs

```diff
@@ -14,35 +14,43 @@
     </div>
   {{/if}}
 
-  <h1 data-test-title class="title with-headroom">
-    Allocation {{model.name}}
-    <span class="bumper-left tag {{model.statusClass}}">{{model.clientStatus}}</span>
-    <span class="tag is-hollow is-small no-text-transform">
-      {{model.id}}
-      {{copy-button clipboardText=model.id}}
-    </span>
-    {{#if model.isRunning}}
-      {{two-step-button
-        data-test-stop
-        idleText="Stop"
-        cancelText="Cancel"
-        confirmText="Yes, Stop"
-        confirmationMessage="Are you sure? This will reschedule the allocation on a different client."
-        awaitingConfirmation=stopAllocation.isRunning
-        disabled=(or stopAllocation.isRunning restartAllocation.isRunning)
-        onConfirm=(perform stopAllocation)}}
-      {{two-step-button
-        data-test-restart
-        idleText="Restart"
-        cancelText="Cancel"
-        confirmText="Yes, Restart"
-        confirmationMessage="Are you sure? This will restart the allocation in-place."
-        awaitingConfirmation=restartAllocation.isRunning
-        disabled=(or stopAllocation.isRunning restartAllocation.isRunning)
-        onConfirm=(perform restartAllocation)}}
-    {{/if}}
+  <h1 data-test-title class="title with-headroom with-flex">
+    <div>
+      Allocation {{model.name}}
+      <span class="bumper-left tag {{model.statusClass}}">{{model.clientStatus}}</span>
+    </div>
+    <div>
+      {{#if model.isRunning}}
+        <div class="two-step-button">
```

Comment:
I considered having a duplication of this class called something like `.like-two-step-button` because this isn’t an actual two-step button but using this class made it align properly with the other buttons in this section. It seemed maybe unnecessary…?

Reply:
Yeah, it's fine for now. I think the entire way we handle buttons in titles needs a larger rethink.